packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/index.ts

```diff
@@ -362,36 +365,41 @@ class IssuePrepaidCardWorkflowComponent extends Component {
   @service declare workflowPersistence: WorkflowPersistence;
   @service declare router: RouterService;
 
-  workflow!: IssuePrepaidCardWorkflow;
+  @tracked workflow: IssuePrepaidCardWorkflow | null = null;
 
   constructor(owner: unknown, args: {}) {
     super(owner, args);
 
-    const workflow = new IssuePrepaidCardWorkflow(getOwner(this));
-    const persistedState = workflow.session.getPersistedData()?.state ?? {};
-    const willRestore = Object.keys(persistedState).length > 0;
+    let workflow = new IssuePrepaidCardWorkflow(getOwner(this));
```

Comment:
I tend to use `const` as a default for everything but it looks like the convention here is to use `let`. I'm fine with either tbh, will remember to use `let` from now on.

Reply:
Confirmed on let over const for locals. This preference in much of Ember community originated here: https://madhatted.com/2016/1/25/let-it-be.